File: E-Commerce-Project-Xamarin/ShopProject/ShopProject/ViewModel/Order.cs
using System; // Add this using directive

public class Order
{
    public long Id { get; set; }
    public string CommandeId { get; set; }
    public DateTime? DateCommande { get; set; }
    public long? ProductId { get; set; }
    public int? Prix { get; set; }
    public int? Quantite { get; set; }
    public long? UserId { get; set; }
    public string Statue { get; set; }
    public string TypePayment { get; set; }
    public string Adresse { get; set; }
    public Product Product { get; set; } // Assuming you have a Product class
}

File: E-Commerce-Project-Backend/src/main/java/org/sid/ecommerce/Entities/Order.java
package org.sid.ecommerce.Entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@Entity
@Table(name = "`order`")
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Order {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private String CommandeId;
    private Date DateCommande;
    @ManyToOne
    private Product Product;
    private int prix;
    private int quantite;
    @ManyToOne
    private User User;
    private String Statue;
    private String typePayment;
    private String adresse;
}